### ISSUE TITLE
Vztah mezi typem pohledu a vybranými slovníky

### ISSUE THREAD

**Issue by jakubklimek** [datagov-cz/sgov-assembly-line#175]
# Popis problému
Není jasné, proč změna pohledu z kompaktního na plný má vliv na slovníky zobrazené vlevo

![přepnutí-pohledů](https://github.com/datagov-cz/sgov-assembly-line/assets/718827/e16b4c6c-1f1f-49c6-80be-82e1ac256247)

# Závažnost problému
4. příliš mojí práci neovlivňuje.

**Comment by bindeali** [comment]
To není bug, vyplývá to z toho, že v kompaktním pohledu nejsou zobrazeny typy vztahů jako pojmy, ale jako vazby, takže nemá smysl je zobrazovat v panelu slovníků a pojmů.

**Comment by jakubklimek** [comment]
Jasně, chápu. Intuitivně mě to ale mate, protože bych čekal, že vlevo mám pohled na pojmy které používám a vpravo jen vybírám jak je zobrazit, což by na to nemělo mít vliv. Jestliže tím přepínám nejen typ zobrazení, ale i způsob interpretace toho co je a co není pojem, tak to je samozřejmě jiná. Teď nevim přesně co s tim, ale stále mi to přijde trochu matoucí.

**Comment by MichalMed** [comment]
Jak píše Alice, tohle není bug ale featura. Je to z toho důvodu, že podle zkušeností většina uživatelů nepovažuje vazby a vlastnosti za pojmy. Defaultní mód je kompaktní, ve kterém typy vazeb nejsou zobrazovány jako pojmy ani v kanvasu, ani v seznamu. Pro fajnšmekry je full mode.

**Comment by jakubklimek** [comment]
Jo, to souhlasim. Ja jen rikam ze mi to prijde matouci. Ale priznavam ze nevim co s tim :)

**Comment by bindeali** [comment]
#180 je naprosto stejný problém se stejným odůvodněním. Řešení vidím v úpravě UX, aby to bylo jasnější.
